### ISSUE TITLE
not possible to update regular cursor color after ryo initializes

### ISSUE THREAD

**Issue by mattly** [Kungsgeten/ryo-modal#64]
I might be missing something, but it appears that ryo looks at the current theme's cursor color when it initializes. If I change the theme (as I often do during a session, between light & dark mode), the regular-mode cursor stays with the initial theme's cursor color.

I can update the modal cursor color programatically, as this is a var, but the regular cursor color that ryo uses is a const.

**Comment by Kungsgeten** [comment]
Hi! I guess I could change it to defvar instead of defconst. Do you intend to update the variable after you change themes?

**Comment by mattly** [comment]
yeah, I have an after-theme-change hook to take care of a few other things like the modeline, and could just add it to that
